components/NavigationComponent.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Component } from 'react'
+import { AppRegistry, Text, View } from 'react-native'
+import { connect } from 'react-redux'
+
+import { next, tutorial, menu } from '../actions'
+
+class NavigationComponent extends Component {
+  tutorial() {
+    console.log(this.props.tutorialId)
+    if(this.props.tutorialId != undefined) {
+      return <Text onPress={ () => this.props.tutorial(this.props.curViewId) }>Tutorial</Text>
+    }
+  }
+
+  render() {
+    return <View style={{
```

Comment:
also: if you split a component into multiple lines, the structure becomes clearer by placing `>` on its own line:
```
<View
  foo="bar"
  bar="foo"
>
  {children}
</View>
```